### ISSUE TITLE
Multivariant kalibrering

### ISSUE THREAD

**Issue by anjamaa** [realfagstermer/realfagstermer#1235]
Skal denne være "Multivariat [...]"? Ser sånn ut på katalogpostene. Har oversatt den til `Multivariate calibration` :)

---
*REAL000667 (Multivariant kalibrering) i [Emnesøk](https://app.uio.no/ub/emnesok/realfagstermer/search?id=c000667), [Skosmos](http://data.ub.uio.no/realfagstermer/c000667), [Okapi](http://ub-viz01.uio.no/okapi2/#/search?q=realfagstermer%3A%22Multivariant%20kalibrering%22), [Soksed](http://ub-soksed.uio.no/concepts/REAL000667)*

**Comment by BioHeidi** [comment]
Ja, endrer denne på norsk og nynorsk. Kan evt beholde varianten "_-variant_" som synonym, men tror ikke det er mye brukt.
